tools/go-cli/go-whisk/whisk/shared.go

```diff
@@ -38,4 +38,5 @@ type Parameters *json.RawMessage
 type Limits struct {
     Timeout int `json:"timeout,omitempty"`
     Memory  int `json:"memory,omitempty"`
+    Logsize int `json:"logs"`
```

Comment:
`omitempty` is not feasible though, it will turn a `0` (perfectly valid) to an empty object. Have a solution for that?